tests/functional/glusterd/test_glusterd_start_stop_output.py

```diff
@@ -0,0 +1,90 @@
+#  Copyright (C) 2021 Red Hat, Inc. <http://www.redhat.com>
+#
+#  This program is free software; you can redistribute it and/or modify
+#  it under the terms of the GNU General Public License as published by
+#  the Free Software Foundation; either version 2 of the License, or
+#  any later version.
+#
+#  This program is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#
+#  You should have received a copy of the GNU General Public License along
+#  with this program; if not, write to the Free Software Foundation, Inc.,
+#  51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+
+"""Test Description:
+    Consistent info display while starting and stopping glusterd.
+"""
+
+from time import sleep
+from glusto.core import Glusto as g
+from glustolibs.gluster.gluster_base_class import GlusterBaseClass, runs_on
+from glustolibs.gluster.exceptions import ExecutionError
+from glustolibs.gluster.gluster_init import (restart_glusterd,
+                                             wait_for_glusterd_to_start)
+
+
+@runs_on([['replicated', 'distributed', 'arbiter', 'dispersed',
+           'distributed-replicated', 'distributed-arbiter',
+           'distributed-dispersed'], ['glusterfs']])
+class TestGlusterdStartStopOutput(GlusterBaseClass):
+    def setUp(self):
```

Comment:
No need to override baseclass setup in this context.

Reply:
Ack.